apache/trafficcontrol
# deliveryservices/id/capacity no longer gives Internal Server Error

## What does this PR (Pull Request) do?


- [x] This PR fixes #4237 


## Which Traffic Control components are affected by this PR?


- Traffic Ops

## What is the best way to verify this PR?

1. Run CIAB
2. Open Traffic Portal, create a delivery service (ds).
3. Run https://TP-url:port/api/3.0/deliveryservices/ds-id/capacity
Note: You should see the following response:
```
{"response":
    {
     "availablePercent":0,
     "unavailablePercent":0,
     "utilizedPercent":0,
     "maintenancePercent":0
    }
}
```

## If this is a bug fix, what versions of Traffic Control are affected?


- master 00294da

## The following criteria are ALL met by this PR




## Additional Information

Documentation has a correct example.
